### ISSUE TITLE
Different masks used in communities activities list

### ISSUE THREAD

**Issue by brettski** [ThatConference/that.us#493]
As you can see from the image the "what will you create" image is masked differently than the session one. I assume they are using different settings. Ironically I am not sure which is the correct way. A: my head is narrow and tall (maybe normal 🤔) B: I am certainly squashed. 

![image](https://user-images.githubusercontent.com/473633/98756011-4d5a6f80-238f-11eb-8bc7-f6e26f0610c8.png)

**Comment by asharonbaltazar** [comment]
Where can I reproduce this bug?

**Comment by brettski** [comment]
It can't currently be reproduced as a quick workaround done on the community pages was to replace the members image with a large "➕" 

I am not sure on @csell5 thoughts, but I am thinking we can leave this one as is now and close this. 

Current view on community pages (e.g. /communities/javascript)
![image](https://user-images.githubusercontent.com/473633/106040025-7c83a180-609f-11eb-8078-f1c804180570.png)

**Comment by csell5** [comment]
I'm guessing this is just something simple. The two components are FeaturedActivity-Add.svelte and FeaturedActivity.svelte

It's looking like the image crop we place on it isn't aligned between the two.

```
    <img
      class="mx-auto h-40 w-40 rounded-full xl:w-56 xl:h-56"
      src="{`${$thatProfile.profileImage}${imageCrops.profile}`}"
      alt="{`${$thatProfile.firstName} ${$thatProfile.lastName}`}"
    />   
```

vs

```
    <img
      class="mx-auto h-40 w-40 rounded-full xl:w-56 xl:h-56"
      src="{userProfileImage}"
      alt="{`${host.firstName} ${host.lastName}`}"
    />
```

with an image crop of '?mask=ellipse&w=500&h=500&fit=crop'

We'd need to look into the data getting passed in, but it should be pretty simple.

**Comment by asharonbaltazar** [comment]
Any dummy account I can use so I can reproduce these issues?

**Comment by csell5** [comment]
So we don't have a proper dev env setup yet. But you can just create a test activity (just mark it testing) and add it to a community by tagging @[community], then when you're done go back and cancel it.

**Comment by asharonbaltazar** [comment]
They seem to be the same for me... What am I doing wrong?

![image](https://user-images.githubusercontent.com/58940073/106218319-b4bcda00-619c-11eb-80c5-2f45810e5cb0.png)

**Comment by csell5** [comment]
@brettski do you have any more info that could help us?

**Comment by brettski** [comment]
> @brettski do you have any more info that could help us? @brettski or maybe I can checkout your logrocket session if you can make it happen.

I do not. I assume the original issue was due to the masking being different between the two. Perhaps it is the same now as things have been cleaned up. If it looks correct, lets roll with it.

**Comment by csell5** [comment]
ok, let's put it on ice then. I did fix the masks sometime ago https://github.com/ThatConference/that.us/pull/527 so maybe this issue just got missed in getting associated.

**Comment by csell5** [comment]
@asharonbaltazar thoughts?

**Comment by asharonbaltazar** [comment]
I mean, if it's resolved, why keep this issue open? If you want, I can remove that icon and the associated `hover` behavior along with it.

**Comment by brettski** [comment]
> ok, let's put it on ice then. I did fix the masks sometime ago #527 so maybe this issue just got missed in getting associated.

I am not sure what you mean by "On Ice", shouldn't the pr go through to add the image back to the "create activity" card?

**Comment by brettski** [comment]
> I mean, if it's resolved, why keep this issue open? 

Agreed
